partcad/src/partcad/part_factory_cadquery.py

```
@@ -114,12 +126,10 @@
             try:
                 # pc_logging.error("Response: %s" % response_serialized)
                 response = base64.b64decode(response_serialized)
-                register_cq_helper()
+                register_ocp_helper()
                 result = pickle.loads(response)
```

Ruff (0.8.2)

`pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)